c_glib/arrow-glib/table.cpp

```diff
@@ -553,9 +553,9 @@ garrow_table_concatenate(GArrowTable *table,
     arrow_tables.push_back(arrow_other_table);
   }
   std::shared_ptr<arrow::Table> arrow_concatenated_table;
-  auto status = arrow::ConcatenateTables(arrow_tables, &arrow_concatenated_table);
-  if (garrow_error_check(error, status, "[table][concatenate]")) {
-    return garrow_table_new_raw(&arrow_concatenated_table);
+  auto arrow_concatenated_table = arrow::ConcatenateTables(arrow_tables);
```

Comment:
Variable shadowing.

Reply:
@brills Could you remove `std::shared_ptr<arrow::Table> arrow_concatenated_table;`?